test/v2_integration.py

```diff
@@ -41,6 +41,155 @@ def test_wildcardmultidomain():
 def test_http_challenge():
     chisel2.auth_and_issue([random_domain(), random_domain()], chall_type="http-01")
 
+def rand_http_chall(client):
```

Comment:
I think this chunk is all the same as integration-test.py, but s/chisel/chisel2/. So please update after making the fixes above.